setup.py

```diff
@@ -74,36 +74,44 @@ def get_version(version_file):
         # that you indicate whether you support Python 2, Python 3 or both.
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.7",
```

Comment:
My one concern here is that this package is a dependency of [cisagov/cyhy-feeds](https://github.com/cisagov/cyhy-feeds) which itself is deployed in [cisagov/cyhy_amis](https://github.com/cisagov/cyhy_amis) to an instance running on Debian 10 which only (officially) has [Python 3.7.3](https://packages.debian.org/buster/python/python3).

Reply:
Ugh. That is a problem.

Reply:
We could take the same approach as we previously did for 3.6 and have a slightly cursed `include` on the matrix to continue testing for Python 3.7 and I guess keep 3.8 for now at the same time?

Reply:
I'm inclined to just keep support for 3.7 and 3.8 in this repo for now rather than contort ourselves to jump through hoops. I'll redo the merge tomorrow. 

We could look for another non-rolling release distro to use for the CyHy instances. Maybe we could find something that offers Python 2 and a newer version of Python 3 than is offered by Buster. It might be worth looking at what packages are offered by old versions of something like CentOS, for example.

Reply:
Just to be clear the `include` on the matrix will be necessary to maintain support for Python 3.7. It will be like [this one in cisagov/cyhy-feeds](https://github.com/cisagov/cyhy-feeds/blob/2691396f163c9d5edd0866a7fa4073989c62fa6d/.github/workflows/build.yml#L119-L121) to get a runner image version that will support Python 3.7.

Reply:
Well your `if` works just as well 😆